docker-bake.hcl

```
@@ -84,120 +76,95 @@ variable "DEBIAN_RELEASE" {
   default = "bookworm-20240423"
 }
 
-target "alpine_jdk11" {
-  dockerfile = "alpine/Dockerfile"
-  context    = "."
-  args = {
-    ALPINE_TAG   = ALPINE_FULL_TAG
-    JAVA_VERSION = JAVA11_VERSION
-  }
-  tags = [
-    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${VERSION}-alpine-jdk11" : "",
-    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${VERSION}-alpine${ALPINE_SHORT_TAG}-jdk11" : "",
-    "${REGISTRY}/${JENKINS_REPO}:alpine",
-    "${REGISTRY}/${JENKINS_REPO}:alpine-jdk11",
-    "${REGISTRY}/${JENKINS_REPO}:latest-alpine-jdk11",
-    "${REGISTRY}/${JENKINS_REPO}:alpine${ALPINE_SHORT_TAG}",
-    "${REGISTRY}/${JENKINS_REPO}:alpine${ALPINE_SHORT_TAG}-jdk11",
-    "${REGISTRY}/${JENKINS_REPO}:latest-alpine${ALPINE_SHORT_TAG}-jdk11",
-    "${REGISTRY}/${JENKINS_REPO}:latest-alpine${ALPINE_SHORT_TAG}",
-  ]
-  platforms = ["linux/amd64"]
+variable "default_jdk" {
+  default = 17
```

```suggestion
  default = 11
```
Setting the default JDK to 11 so we can introduce this pull request without any (breaking) change.

Solving #399 will then be as simple as setting this default_jdk value to `17`.

---

Compared to the changes needed in https://github.com/jenkinsci/docker-ssh-agent/pull/400/files, this illustrates the benefit of this PR.